src/components/PlaylistPage/LessonItem.scss

```diff
@@ -32,3 +32,21 @@
 .progress {
   opacity: 0.5;
 }
+
+.marginLeft {
+  margin-left: 7px !important;
```

Comment:
FY! Ikke bruk `!important`. Finn heller en annen måte å ikke la `.glyph` sin `margin-left: auto` ta overhånd... kanskje ting burde struktureres litt annerledes? Som "siste utvei" kan du alltids øke spesifisiteten på `.marginLeft` ved å skrive det to ganger:
```
.marginLeft.marginLeft {
  margin-left: 7px;
}
```

Reply:
Hva med å ta vekk `margin-left` fra `externalGlyph` og `glyph`, og heller ha 2 forskjellige klasser `.marginLeftAuto { margin-left: auto; }` og `.marginLeftShort { margin-left: 7px; }`?